TombRaiderII/ComponentSettings.cs

```diff
@@ -35,18 +39,20 @@ private void InitializeComponent()
             _modeSelect.TabIndex = 0;
             _modeSelect.TabStop = false;
             _modeSelect.Text = "Mode Selection";
-
+            // 
             // ILModeButton
+            // 
             ILModeButton.AutoSize = true;
             ILModeButton.Location = new System.Drawing.Point(84, 20);
             ILModeButton.Name = "ILModeButton";
-            ILModeButton.Size = new System.Drawing.Size(135, 17);
+            ILModeButton.Size = new System.Drawing.Size(139, 17);
             ILModeButton.TabIndex = 1;
-            ILModeButton.Text = "Individual Levels (RTA)";
+            ILModeButton.Text = "IL or Section Run (RTA)";
             ILModeButton.UseVisualStyleBackColor = true;
-            ILModeButton.CheckedChanged += ILModeButtonCheckedChanged;
-
+            ILModeButton.CheckedChanged += new EventHandler(ILModeButtonCheckedChanged);
```

Comment:
I changed this back to the original along with the other 3 event viewers and things work the exact same way, so I think these should stay as they were.